apps/mockingbird/mbird.py

```diff
@@ -0,0 +1,905 @@
+The program can also be used to change the SmartServer to Independent Management Mode (IMM) which is needed for MBird to run. 
+
+
+Variables to be supplied by user:
+datapointin - The name of the input datapoint to reflect
+datapointout - The name of the output datapoint to reflect to
+NOTE: These values are used for the example 2)
+
+incrementlist - The list of datapoints to be increment once every time period defined by message_wait
+"""
+
+
+# Update Lamp/0/nviLamp on a Mockingbird device from the SmartServer and you get the event below (whether you enable monitoring or not).  So
+# we will look for "updated" in the topic and then look for "value": in the payload
+# glp/0/17thbp9/ev/updated/dev/lon/type/9FFFFF0501840460 {"handle":"m-01","topic":"glp/0/17thbp9/fb/dev/lon/m-01","block":"if/Lamp/0","datapoint":"nviLamp","value":{"value":87,"state":0},"health":"normal"}
+# And then we want to write this value back to an output.  Here is an example:
+# glp/0/17thbp9/rq/dev/lon/m-01/if/Switch/0 {"nvoSwitch":{"value":{"value":98,"state":1},"prio":17}}
+
+# This is what I got from the program below after setting the input to 53 / 1:
+# glp/0/17thbp9/rq/dev/lon/m-01/if/Switch/0 {"nvoSwitch":{"value":{"value":53,"state":0},"prio":17}}
+# glp/0/17thbp9/rq/dev/lon/m-01/if/Switch/0 {"nvoSwitch":{"value":{"value": 54, "state": 0},"prio":17}}
+# glp/0/17thbp9/rq/dev/lon/m-01/if/Switch/0 {"nvoSwitch":{"value":{"value":33,"state":1},"prio":17}}
+
+
+# Message when connected to the SS
+# TOPIC: glp/0/17qeiqj/fb/dev/lon/dyn-1-01/cfg
+# {"name":"dyn-1-01","desc":"Internal device applications","loc":{"desc":"","lat":0,"lng":0,"ele":0,"row":0,"column":0,"sunrise":-6,"sunset":-6},"motion_zone":0,"motion_radius":0,"motion_timeout":0,"mru":"2023-09-13 14:01:33.381 UTC"}
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~ USER CONFIGURABLE SETTINGS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Filenames to store the newly created MBirds.
+scr_filename = 'nuDevs.scr'
+csv_filename = 'example_devices.csv'
+
+message_wait = 0.1  # Delay between datapoint updates.
+SERVICE_PIN_WAIT = 1  # Time between two service pin attempts.
+SERVICE_PIN_TRYS = 5  # Number of attempts before giving up.
+INITIAL_WAIT_TIME = 5  # Dead delay for all messages to come.
+MESSAGE_TIMEOUT = 2 # Query message timeout
+DEVICE_CREATION_TIMEOUT = 30
+MAX_LEP_ID = 30 # Maximum LEP ID when searching for the interface. 
+
+# Datapoints used for mirroring.
+# If datapointin changes, the datapointout will also be changed.
+datapointin = "Lamp/0/nviLamp"
+datapointout = "Lamp/0/nvoLampFb"
+
+# Datapoints to be incrementally updated:
+datapoints = ["Lamp/1/nviLamp", "Lamp/1/nvoLampFb", "Switch/0/nviSwitchFb",
+              "Switch/1/nviSwitchFb", "Switch/0/nvoSwitch", "Switch/1/nvoSwitch"]
+
+# You can use this list to set the device health of individual MBrids.
+# Create an list of tuples with exact device handle and the desired device health.
+dev_state_list = None
```

Comment:
Similar comment as in readme.  Not clear whether "true" means "up" or "down".

Reply:
We're effectively enabling the devices, so I'll change the variable names accordingly and leave the true = power on, false = power off notation.